Adyen.xcodeproj/project.pbxproj

```diff
@@ -4975,7 +4975,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 1;
+				CURRENT_PROJECT_VERSION = 2;
```

Comment:
What is this number based on?

Reply:
its because Xcode doesn't accept the beta format, so we keep the version `4.0.0` in Xcode, but increment the build number, hence `2` instead of `1` :-)